docs/js.md

```diff
@@ -7,13 +7,12 @@ sidebar_label: Scala.js
 Code fences with the `scala mdoc:js` modifier are compiled with Scala.js and run
 in the browser.
 
-```scala mdoc:js:shared:invisible
```

Comment:
In fairness, this should probably have been proposed in a seperate change. There's no reason to do it as part of this PR. 

I have a personal preference for showing imports being used - I think it makes it clearer to the reader. However, I agree this could have been seperated into another PR. I'm happy to do that if it's preferred.